examples/unstructured_2d_dgsem/elixir_euler_basic.jl

```diff
@@ -58,12 +58,16 @@ save_solution = SaveSolutionCallback(interval = 10,
 
 stepsize_callback = StepsizeCallback(cfl = 0.9)
 
+time_series = TimeSeriesCallback(semi, [(2.0, -0.5), (0.28, 1 / 3), (1.87, 2 / 3)];
+                                 interval = 10)
+
 callbacks = CallbackSet(summary_callback,
                         analysis_callback,
                         alive_callback,
                         save_restart,
                         save_solution,
-                        stepsize_callback)
+                        stepsize_callback,
+                        time_series)
```

Comment:
```suggestion
                        time_series,
                        stepsize_callback)
```

For consistency with our documentation - we first save data before we apply callbacks belonging to the next step.

Reply:
I need to change which elixir uses this new callback anyway because now some of the other tests that use this elixir fail. But I will keep this in mind when i modify a different elixir.